### PR TITLE
feat: record dubbo request param type

### PR DESCRIPTION
**Pull request by lucas-myx**

### CODE REVIEW

File: arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java [hunk 1]
```diff
@@ -59,11 +60,24 @@ public String getServiceOperation() {
...
+        return parseRequest(invocation.getArguments(), Serializer::serialize);
+    }
+    public String getRequestParamType() {
+        Class<?>[] parameterTypes = invocation.getParameterTypes();
+        if (parameterTypes != null && parameterTypes.length > 0) {
+            return parameterTypes[0].getName();
+        }
+        return parseRequest(invocation.getArguments(), TypeUtil::getName);
+    }
+    public static String parseRequest(Object request, Function<Object, String> parser) {
```
Comment (mokdeve) [inline arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java:72]: >  public static String parseRequest

Would you change "public" to internal access level?

**Review by mr3 — APPROVED** [review]

**Review by YongwuHe — APPROVED** [review]

File: arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java [hunk 2]
```diff
@@ -59,11 +60,24 @@ public String getServiceOperation() {
         return getPath() + "." + getOperationName();
     }
     public String getRequest() {
-        Object request = null;
-        if (invocation.getArguments() != null && invocation.getArguments().length > 0) {
-            request = invocation.getArguments()[0];
+        return parseRequest(invocation.getArguments(), Serializer::serialize);
```
Comment (mokdeve) [inline arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java:63]: >return parseRequest(invocation.getArguments(), Serializer::serialize);

How to determine which serializer is supported to use?

The serializer used by the user may be customized and not supported, such as JSON.

File: arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java (hunk 1 above)
Comment (lucas-myx) [reply arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java:72]: DubboStreamAdapter#getRequest also use

File: arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java (hunk 2 above)
Comment (lucas-myx) [reply arex-instrumentation/dubbo/arex-dubbo/src/main/java/io/arex/inst/dubbo/DubboAdapter.java:63]: customize serializer protocol this suite situation only arex-schedule replay need (specify the serialization implemet for the user through SPI during replaying)

**Pull request merged**